assets/src/stories-editor/components/higher-order/with-amp-story-settings.js

```diff
@@ -307,6 +309,8 @@ export default createHigherOrderComponent(
 
 								stopBlockActions();
 							} }
+							snap={ [ -165, -135, -120, -105, -90, -75, -60, -45, -30, -15, 0, 15, 30, 45, 60, 75, 90, 105, 120, 135, 150, 165, 180 ] }
```

Comment:
Could we create a constant for this, or move this to `constants.js`? Seems to be identical for both lines where used.

Or potentially even create a function which given the difference of the degrees (e.g. 15deg as it is here) returns an array. In case we'd want to change it to 10 or 20 for example. Although that's likely an overkill until we actually need it :)

Reply:
I was thinking about passing a function as well.

Both options work for me though.

Reply:
We might later consider having a different degree step for tiny blocks versus larger blocks for example.

Reply:
Added constants for these.